.github/workflows/bbw_build_container_rhel.yml

```diff
@@ -155,35 +182,59 @@ jobs:
           else
             echo "Not pushing images to registry"
           fi
+
       - name: Login to ghcr.io
         if: ${{ env.DEPLOY_IMAGES == 'true' }}
         uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Push images to ghcr.io
-        if: ${{ env.DEPLOY_IMAGES == 'true' }}
+
+      - name: ghcr.io - push dev tag
+        if: ${{ env.DEPLOY_IMAGES == 'true' && env.MAIN_BRANCH == 'false' }}
         run: |
           msg="Push docker image to ghcr.io (${{ env.IMG }})"
           line="${msg//?/=}"
           printf "\n${line}\n${msg}\n${line}\n"
           skopeo copy --all --src-tls-verify=0 \
             docker://localhost:5000/${{ env.REPO }}:${{ env.IMG }} \
+            docker://ghcr.io/${GITHUB_REPOSITORY,,}/${{ env.REPO }}:dev_${{ env.IMG }}
+
+      - name: ghcr.io - move tag to production
+        if: ${{ env.DEPLOY_IMAGES == 'true' && env.MAIN_BRANCH == 'true' }}
+        run: |
+          msg="Push docker image to ghcr.io (${{ env.IMG }})"
```

Comment:
the message should probably be adapted (it's not a "push" but a kind of "move"?)

Reply:
@fauust  Oh, forgot about this. Thanks!